src/help/components/entry/__tests__/Entry.spec.tsx

```diff
@@ -15,6 +15,11 @@ const mock = new MockAdapter(axios);
 
 mock.onGet(/api\/help\/canonical_and_isoforms/).reply(200, mockData);
 
+jest.mock('../../../../shared/components/layouts/UniProtFooter', () => ({
+  __esModule: true,
+  default: () => '{{ Footer }}',
+}));
+
```

Comment:
Mock footer to avoid filling up the snapshot with the footer's content which is already tested somewhere else so would mainly be noise for the test here.